command/shell_test.go

```diff
@@ -55,6 +55,18 @@ func (s *shellExecuteCommandSuite) TestWorksWithEmptyShell() {
 	s.NoError(cmd.Execute(s.ctx, s.comm, s.logger, s.conf))
 }
 
+
+func (s *shellExecuteCommandSuite) TestErrorToIgnoreAndRedirectToStdOut() {
+	cmd := &subprocessExec{}
+	s.NoError(cmd.ParseParams(map[string]interface{}{}))
+
+	cmd.Silent = true
+	s.NoError(cmd.ParseParams(map[string]interface{}{}))
+
+	cmd.RedirectStandardErrorToOutput = true
+	s.Error(cmd.ParseParams(map[string]interface{}{}))
```

Comment:
Could you instead test that `cmd.IgnoreStandardError` and `cmd. IgnoreStandardOutput` are true? I would have imagined the test patterns here is
1. Assert that `cmd.IgnoreStandardError` and `cmd. IgnoreStandardOutput` are false.
2. Set `cmd.Silent = true`.
3. Execute `cmd.ParseParams`.
3. Assert that `cmd.IgnoreStandardError` and `cmd. IgnoreStandardOutput` are true.

Reply:
Definitely a better flow, my mistake!